scipy/sparse/linalg/isolve/lsqr.py

```diff
@@ -408,30 +406,33 @@ def lsqr(A, b, damp=0.0, atol=1e-8, btol=1e-8, conlim=1e8,
     # Main iteration loop.
     while itn < iter_lim:
         itn = itn + 1
-        """
-        %     Perform the next step of the bidiagonalization to obtain the
-        %     next  beta, u, alfa, v.  These satisfy the relations
-        %                beta*u  =  a*v   -  alfa*u,
-        %                alfa*v  =  A'*u  -  beta*v.
-        """
+        # Perform the next step of the bidiagonalization to obtain the
+        # next  beta, u, alfa, v. These satisfy the relations
+        #     beta*u  =  a*v   -  alfa*u,
+        #     alfa*v  =  A'*u  -  beta*v.
```

Comment:
Certainly seems like a sensible change to avoid creating the strings like that yeah. I wonder if it might even improve performance, if only microscopically.